### PR TITLE
autots 0.6.15 :snowflake:

### PR DESCRIPTION
**Pull request by psteyer**
autots 0.6.15 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-4741]
- dev_url:        https://github.com/winedarksea/AutoTS/tree/0.6.14
- conda_forge:    https://github.com/conda-forge/autots-feedstock
- pypi:           https://pypi.org/project/autots/0.6.15
- pypi inspector: n/a

### Explanation of changes:

- new version number

[PKG-4741]: https://anaconda.atlassian.net/browse/PKG-4741?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,44 +7,71 @@ package:
...
+  url: https://github.com/winedarksea/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 6811694c415abbeca8eba13c017cfc366d3b060f4873cf39cbaa4ea8a358ac57
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps -vv --no-build-isolation
+  # skipping s390x due to test dependencies not being on platform
+  skip: true  # [py<37 or s390x]
```
Comment (msentissi) [inline recipe/meta.yaml]: Should this be `# [py<36]` ?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -7,44 +7,71 @@ package:
...
+    - python
+    - setuptools
+    - wheel
   run:
-    - python >=3.6
+    - python
     - numpy >=1.14.6
-    - pandas >=0.25.*
+    - pandas >=0.25.0
     - scikit-learn >=0.20.*
```
Comment (msentissi) [inline recipe/meta.yaml]: This is `0.20.0` in the `pyproject.toml`

File: recipe/meta.yaml [hunk 3]
```diff
@@ -7,44 +7,71 @@ package:
...
     - statsmodels >=0.10.*
```
Comment (msentissi) [inline recipe/meta.yaml]: I think this should be `0.10.0`

File: recipe/meta.yaml [hunk 4]
```diff
@@ -7,44 +7,71 @@ package:
...
-    - requests
+
```
Comment (msentissi) [inline recipe/meta.yaml:35]: I see some version pinnings in the `additional` extra. I think we should add them in the `run_constrained` section:

```suggestion
  run_constrained:
    - holidays >=0.9
    - prophet >=0.4.0
    - xgboost >=1.6
    
```

File: recipe/meta.yaml [hunk 5]
```diff
@@ -7,44 +7,71 @@ package:
...
+{% set tests_to_skip = tests_to_skip + " or test_date_part" %} 
+# ValueError: Shape of passed values is (6, 18), indices imply (6, 9)
+{% set tests_to_skip = tests_to_skip + " or test_event_risk" %} 
 
 test:
+  source_files:
+    - tests
   imports:
     - autots
     - autots.datasets
```
Comment (msentissi) [inline recipe/meta.yaml:55]: I'm wondering whether we can also import `autots.evaluator/models/templates/tools` ?

File: recipe/meta.yaml (hunk 2 above)
Comment (psteyer) [reply recipe/meta.yaml:29]: For version `0.6.15`, I'm seeing `0.25.0` for pandas

https://github.com/winedarksea/AutoTS/blob/6e473e89957c69446355d53118f65502840f4c94/pyproject.toml#L22

File: recipe/meta.yaml (hunk 5 above)
Comment (psteyer) [reply recipe/meta.yaml:55]: I can get those added!

File: recipe/meta.yaml (hunk 4 above)
Comment (psteyer) [reply recipe/meta.yaml:35]: I'll get these added!

**Review by skupr-anaconda — APPROVED** [review]

**Review by msentissi — APPROVED** [review]

**Pull request merged**